hydra-plutus/src/Hydra/Contract/Util.hs

```diff
@@ -60,3 +60,8 @@ infix 4 ===
 (===) val val' =
   serialiseData (toBuiltinData val) == serialiseData (toBuiltinData val')
 {-# INLINEABLE (===) #-}
+
+-- | Convert Cardano.Api 'NetworkId' to ledger `Network`
+networkIdToNetwork :: NetworkId -> Network.Network
+networkIdToNetwork Mainnet = Network.Mainnet
+networkIdToNetwork (Testnet _) = Network.Testnet
```

Comment:
Let's not make a huge `Util` module just for all kinds of stuff. This function is currently only used in `Hydra.Contract.Commit` so let's put it there (into the where clause).. or just see my other comment.